Documentation/html_viewer/js/dictionary_functions.js

```diff
@@ -165,6 +165,18 @@
 		}
 	}
 	
+	function sortedKeys(array)
+	{
+    	var keys = [];
+
+		for(var key in array)
+		{
+			keys.push(key);
+		}
+
+		return keys.sort();
+	}
```

Comment:
Arrays in Javascript are like LCB's lists -- it might be clearer if you rename the argument to `sortedKeys` from `array` to `obj`.

Did you try writing this as something like the following?

```javascript
function sortedKeys(obj)
{
    return obj.keys().sort();
}
```